packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -26,21 +26,30 @@ export class AtomicSegmentedFacetScrollable {
   }
 
   private renderArrow(arrowDirection: string) {
-    return (
+    return [
       <Button
         style="square-neutral"
-        class="flex shrink-0 basis-8 justify-center items-center rounded"
+        class={`flex shrink-0 basis-8 justify-center items-center rounded absolute z-10 w-10 top-0 bottom-0 ${
+          arrowDirection === ArrowLeftIcon ? 'left-0' : 'right-0'
+        }`}
         ariaHidden="true"
         onClick={() => this.horizonalSlider(arrowDirection)}
       >
         <atomic-icon class="w-3.5" icon={arrowDirection}></atomic-icon>
-      </Button>
-    );
+      </Button>,
+      <div
+        class={`fade w-16 h-10 absolute top-0  z-[5] from-white-80 ${
+          arrowDirection === ArrowLeftIcon
```

Comment:
Same as mentioned in the other PR, I would not take the icon as a value for `arrowDirection`.